raiden/tests/scenarios/bf2_long_running.yaml

```diff
@@ -0,0 +1,332 @@
+          tasks:
+            - wait: 30
+      - parallel:
+          name: "Verify if channels were opened correctly"
+          tasks:
+            - assert: {from: 0, to: 1, total_deposit: 0, balance: 0,  state: "opened"}
+            - assert: {from: 1, to: 0, total_deposit: 0, balance: 0,  state: "opened"}
+            - assert: {from: 1, to: 2, total_deposit: 0, balance: 0,  state: "opened"}
+            - assert: {from: 2, to: 1, total_deposit: 0, balance: 0,  state: "opened"}
+            - assert: {from: 2, to: 3, total_deposit: 0, balance: 0,  state: "opened"}
+            - assert: {from: 3, to: 2, total_deposit: 0, balance: 0,  state: "opened"}
+      - parallel:
+          name: "0 1 2 3 make a deposit of 100 to their channels"
+          tasks:
+            - deposit: {from: 0, to: 1, total_deposit: 100_000_000_000_000_000}
+            - deposit: {from: 1, to: 0, total_deposit: 100_000_000_000_000_000}
+            - deposit: {from: 1, to: 2, total_deposit: 100_000_000_000_000_000}
+            - deposit: {from: 2, to: 1, total_deposit: 100_000_000_000_000_000}
+            - deposit: {from: 2, to: 3, total_deposit: 101_000_000_000_000_000}
+            - deposit: {from: 3, to: 2, total_deposit: 101_000_000_000_000_000}
+      - parallel:
+          name: "Checking if the deposit was successful"
+          tasks:
+            - assert: {from: 0, to: 1, total_deposit: 100_000_000_000_000_000, balance: 100_000_000_000_000_000, state: "opened"}
+            - assert: {from: 1, to: 0, total_deposit: 100_000_000_000_000_000, balance: 100_000_000_000_000_000, state: "opened"}
+            - assert: {from: 1, to: 2, total_deposit: 100_000_000_000_000_000, balance: 100_000_000_000_000_000, state: "opened"}
+            - assert: {from: 2, to: 1, total_deposit: 100_000_000_000_000_000, balance: 100_000_000_000_000_000, state: "opened"}
+            - assert: {from: 2, to: 3, total_deposit: 101_000_000_000_000_000, balance: 101_000_000_000_000_000, state: "opened"}
+            - assert: {from: 3, to: 2, total_deposit: 101_000_000_000_000_000, balance: 101_000_000_000_000_000, state: "opened"}
+      - parallel:
+          name: "Deposit to a channel that does not exist"
+          tasks:
+            # Node tries to deposit to an non-existing channel
+            - deposit: {from: 4, to: 3, total_deposit: 20_000_000_000_000_000, expected_http_status: 409}
+            # Node tries to deposit less than the total deposit in two of his channels
+            - deposit: {from: 1, to: 0, total_deposit: 50_000_000_000_000_000, expected_http_status: 409}
+            - deposit: {from: 1, to: 2, total_deposit: 50_000_000_000_000_000, expected_http_status: 409}
+            # Opening a channel, but the channel already exists
+            - open_channel: {from: 1, to: 0, total_deposit: 100_000_000_000_000_000, expected_http_status: 409}
+      - serial:
+          name: "Give the nodes time to complete the earlier deposits and channel opening"
```

Comment:
This really shouldn't be necessary. I'd probably delete this entire section of waiting for earlier deposits since they tasks above should wait for the API calls to finish anyway.